product_docs/docs/pgd_cli/1/requirements.mdx

```diff
@@ -1,11 +0,0 @@
----
-title: Requirements
```

Comment:
Information present here is necessary and should be near the Configuration steps.